src/components/addToPlaylistButton/AddToPlaylistButton.tsx

```diff
@@ -49,11 +61,13 @@ export const AddToPlaylistButton = ({ videoId }: Props) => {
     setPlaylistsContainingVideo((prevState) => [...prevState, id]);
 
   const { mutate: mutateRemoveFromPlaylist } = useRemoveFromPlaylistMutation(
-    (id) => checkPlaylistForVideo(id),
+    () => videoRemovedHotjarEvent(),
```

Comment:
I find this a tiny bit hard to read, maybe we could use an object here instead of two params. Something like

```
useRemoveFromPlaylistMutation({
onSuccess: () => videoRemovedHotjarEvent(),
onFailure: (playlistId) => checkPlaylistForVideo(playlistId),
})
```